.github/workflows/test-mobile.yml

```diff
@@ -1,45 +1,143 @@
 name: "[Test] Mobile App"
 
 on:
-  push:
-    branches:
-      - main
-      - develop
-      - release
-      - hotfix
+  # push:
+  #   branches:
+  #     - main
+  #     - develop
+  #     - release
+  #     - hotfix
+  schedule:
+    - cron: "0 2/12 * * *"
```

Comment:
Can you check this cron job @valpinkman

Reply:
will do

Reply:
@ggilchrist-ledger your cron will trigger at 2pm and 2am, if you want noon and midnight => `0 0/12 * * *`